pandas/core/indexes/interval.py

```diff
@@ -1340,7 +1340,7 @@ def _intersection_non_unique(self, other: "IntervalIndex") -> "IntervalIndex":
 
         return self[mask]
 
-    def _setop(op_name, sort=None):
+    def _setop(op_name: str, sort=None):
```

Comment:
Was going to suggest annotating `sort` but looks like unused? Maybe delete altogether?

Can annotate the return here at least as `Callable` would be helpful too (better with subscripting if not too much effort). At first glance I assume `setop` meant that this set some value and wouldn't return anything, but looks not to be the case